i2ccontroller/I2CController.h

```diff
@@ -1,47 +1,34 @@
 #ifndef __I2CCONTROLLER_H__
 #define __I2CCONTROLLER_H__
 
-#include <iostream>
-#include "Compass/HMC6343.h"
-#include "Compass/MockCompass.h"
-#include "AnalogArduino/AR_UNO.h"
-#include "AnalogArduino/MockAnalogArduino.h"
-#include "dbhandler/DBHandler.h"
-#include "thread/SystemState.h"
-#include "utility/Timer.h"
-
-#include <chrono>
-#include <thread>
+
+#include <vector>
+#include <stdint.h>
+#include <wiringPiI2C.h>
+#include <wiringPi.h> // delay
```

Comment:
Usually it implies that that header is only used for the function delay, gives purpose behind the include.